build.gradle

```diff
@@ -351,6 +351,15 @@ task distCompiler(type: Copy) {
     }
 }
 
+task distDef(type: Copy) {
+
+    destinationDir distDir
+
+    from(project("platformLibs").file("src/platform")) {
+        into('klib/platformDef')
+    }
```

Comment:
Are `.sh` scripts included intentionally? I don't expect these to work properly.